spec/20-http_request_header_format.md

```diff
@@ -353,6 +355,6 @@ Vendors receiving a `tracestate` request header MUST send it to outgoing request
 
 Following are allowed mutations:
 
-- **Add a new key/value pair**. The new key/value pair SHOULD be added to the beginning of the list.
+- **Add a new key/value pair**. The new key/value pair SHOULD be added to the beginning of the list. Adding a key/value pair MUST NOT result in the same key being present multiple times.
```

Comment:
This had been called out in "Combined Header Value" already, but it actually belong here, IMO.

Reply:
I like how this formulated. It eliminates concern that if platform doesn't deal with tracestate keys, just propagates it as is - it is not responsible for the duplicates. We may want to spell it out explicitly

Reply:
See this part of my PR comment:

> There seems to be no consensus yet whether an implementation should
> clean up a tracestate value it receives by discarding duplicate keys
> that it does not own. Thus, that aspect is not considered in this
> change.

And my message here https://github.com/w3c/trace-context/issues/446#issuecomment-966451599

TL;DR: I agree with you that this is how it should work (vendors should not need to care about foreign keys at all) but I am not sure if this is consensus, so for now I was reluctant to add this aspect to this PR.

Reply:
I agree that a tracing system should be responsible for deduping their own keys and shouldn't need to analyze or dedupe other keys.

Reply:
I've added that aspect as well now: https://github.com/w3c/trace-context/pull/477/commits/ccbabd6087a3ee9e65889fa54a23f32240436b06